nixos/modules/services/hardware/logiops.nix

```diff
@@ -0,0 +1,147 @@
+in {
+  options.services.logiops = {
+    enable = mkEnableOption
+      "logiops, an unofficial userspace driver for HID++ Logitech devices";
+    package = mkPackageOption pkgs "logiops" {
+      example = "logiops_0_2_3";
+    };
+
+    settings = mkOption {
+      type = types.submodule { freeformType = format.type; };
+      description = ''
+        Configuration for logiops.
+
+        See <https://github.com/PixlOne/logiops/wiki/Configuration> for more details.
+        Also see <https://github.com/PixlOne/logiops/blob/main/logid.example.cfg> for an example config.
+      '';
+      default = { };
+      example = {
+        devices = [{
+          name = "Wireless Mouse MX Master";
+          smartshift = {
+            on = true;
+            threshold = 30;
+            torque = 50;
+          };
+          hiresscroll = {
+            hires = true;
+            invert = false;
+            target = false;
+          };
+          dpi = 1000;
+
+          buttons = [
+            {
+              cid = "0xc3";
+              action = {
+                type = "Gestures";
+                gestures = [
+                  {
+                    direction = "Up";
+                    mode = "OnRelease";
+                    action = {
+                      type = "Keypress";
+                      keys = [ "KEY_UP" ];
+                    };
+                  }
+                  {
+                    direction = "Down";
+                    mode = "OnRelease";
+                    action = {
+                      type = "Keypress";
+                      keys = [ "KEY_DOWN" ];
+                    };
+                  }
+                  {
+                    direction = "Left";
+                    mode = "OnRelease";
+                    action = {
+                      type = "CycleDPI";
+                      dpis = [ 400 600 800 1000 1200 1400 1600 ];
+                    };
+                  }
+                  {
+                    direction = "Right";
+                    mode = "OnRelease";
+                    action = { type = "ToggleSmartshift"; };
+                  }
+                  {
+                    direction = "None";
+                    mode = "NoPress";
+                  }
+                ];
+              };
+            }
+            {
+              cid = "0xc4";
+              action = {
+                type = "Keypress";
+                keys = [ "KEY_A" ];
+              };
+            }
+          ];
+        }];
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.etc."logid.cfg".source =
+      format.generate "logid.cfg" cfg.settings;
+    systemd.services.logiops = {
+      description = "Logitech Configuration Daemon";
+      documentation = [ "https://github.com/PixlOne/logiops" ];
+
+      wantedBy = [ "multi-user.target" ];
+
+      startLimitIntervalSec = 0;
+      after = [ "multi-user.target" ];
+      wants = [ "multi-user.target" ];
```

Comment:
Is there a specific reason why we need this? Usually I thought wantedBy multi-user.target is enough.

Reply:
It is used in the original package, see [this file](https://github.com/PixlOne/logiops/blob/94f6dbab5390c1c7375836dd9314c0c2488e48a3/src/logid/logid.service.in#L4)